specifyweb/frontend/js_src/lib/components/Errors/JSONError.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { jsonStringify } from '../../utils/utils';
+import { className } from '../Atoms/className';
+import { TableIcon } from '../Molecules/TableIcon';
+
+type JSONResponse = {
+  exception: string;
+  message: string;
+  data: any;
+  formattedData: string;
+  traceback: string;
+};
+
+function createJsonResponse(error: string): JSONResponse {

```

Comment:
One of the reasons we follow strictCamelCase convention is that it's unambiguous and consistent. In this case, you had `JSON` and `Json` in the same file. Make sure all usages in the file are `Json` instead of `JSON`

Another reason for this: https://medium.com/@thomasreggi/enforcing-strict-camelcase-5332c53a7484